<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="12" passed="6" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-02T11:43:44 EEST" name="Suite1" finished-at="2024-04-02T11:49:19 EEST" duration-ms="334677">
    <groups>
    </groups>
    <test started-at="2024-04-02T11:43:44 EEST" name="Test1" finished-at="2024-04-02T11:49:19 EEST" duration-ms="334677">
      <class name="FooterTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:45 EEST" name="setUp" finished-at="2024-04-02T11:43:52 EEST" duration-ms="7531" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isEnabledPrivatePolicyTest()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:52 EEST" name="isEnabledPrivatePolicyTest" finished-at="2024-04-02T11:43:53 EEST" duration-ms="202" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isEnabledPrivatePolicyTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:53 EEST" name="tearDown" finished-at="2024-04-02T11:43:53 EEST" duration-ms="352" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:53 EEST" name="setUp" finished-at="2024-04-02T11:43:56 EEST" duration-ms="3582" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isEnabledSiteMapTest()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:56 EEST" name="isEnabledSiteMapTest" finished-at="2024-04-02T11:43:57 EEST" duration-ms="97" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Site map link is not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Site map link is not found
at pages.FooterPage.isEnabledSiteMap(FooterPage.java:55)
at FooterTest.isEnabledSiteMapTest(FooterTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//a[contains(@href,'sitemap') or contains(text(),'Сайтмап') or contains(text(),'Sitemap')]"}
  (Session info: chrome=123.0.6312.86)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [04112621a076993ae30b9ddc539b8e50, findElement {using=xpath, value=//a[contains(@href,'sitemap') or contains(text(),'Сайтмап') or contains(text(),'Sitemap')]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 123.0.6312.86 (9b72c47a0536..., userDataDir: C:\Users\Aguka\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51456}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51456/devtoo..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 04112621a076993ae30b9ddc539b8e50
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:199)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:132)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:51)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:531)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy21.isDisplayed(Unknown Source)
at pages.FooterPage.isEnabledSiteMap(FooterPage.java:53)
... 16 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isEnabledSiteMapTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:57 EEST" name="tearDown" finished-at="2024-04-02T11:43:57 EEST" duration-ms="242" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:43:57 EEST" name="setUp" finished-at="2024-04-02T11:44:01 EEST" duration-ms="3652" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isEnabledTermsAndConditionsTest()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:44:01 EEST" name="isEnabledTermsAndConditionsTest" finished-at="2024-04-02T11:44:01 EEST" duration-ms="50" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Terms and conditions link is not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Terms and conditions link is not found
at pages.FooterPage.isEnabledTermsAndConditions(FooterPage.java:46)
at FooterTest.isEnabledTermsAndConditionsTest(FooterTest.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//a[contains(@href,'terms-of-service') or contains(text(),'Умови користування веб-сайтом') or contains(text(),'Terms of Service')]"}
  (Session info: chrome=123.0.6312.86)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [4593f3d5df4890b2780f061f768682b0, findElement {using=xpath, value=//a[contains(@href,'terms-of-service') or contains(text(),'Умови користування веб-сайтом') or contains(text(),'Terms of Service')]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.86, chrome: {chromedriverVersion: 123.0.6312.86 (9b72c47a0536..., userDataDir: C:\Users\Aguka\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51483}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51483/devtoo..., se:cdpVersion: 123.0.6312.86, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 4593f3d5df4890b2780f061f768682b0
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:199)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:132)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:51)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:531)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy21.isDisplayed(Unknown Source)
at pages.FooterPage.isEnabledTermsAndConditions(FooterPage.java:44)
... 16 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isEnabledTermsAndConditionsTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:44:01 EEST" name="tearDown" finished-at="2024-04-02T11:44:01 EEST" duration-ms="151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:44:01 EEST" name="setUp" finished-at="2024-04-02T11:44:04 EEST" duration-ms="3605" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isPresentSearchInputOnHomePageTest()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:44:04 EEST" name="isPresentSearchInputOnHomePageTest" finished-at="2024-04-02T11:44:04 EEST" duration-ms="61" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isPresentSearchInputOnHomePageTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FooterTest@2ab2710]" started-at="2024-04-02T11:44:04 EEST" name="tearDown" finished-at="2024-04-02T11:44:05 EEST" duration-ms="167" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- FooterTest -->
      <class name="FirstTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:43:45 EEST" name="setUp" finished-at="2024-04-02T11:43:53 EEST" duration-ms="8649" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkH1ShouldBeOneTest()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:43:53 EEST" name="checkH1ShouldBeOneTest" finished-at="2024-04-02T11:46:24 EEST" duration-ms="150864" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/#popup-menu не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/#page-main не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/#popup-mini-cart не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/# не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/blog не дорівнює одному expected [1] but found [0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/#popup-menu не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/#page-main не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/#popup-mini-cart не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/# не дорівнює одному expected [1] but found [2],
Кількість тегів <h1> на сторінці https://piddubnyi.devocean.pro/blog не дорівнює одному expected [1] but found [0]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at FirstTest.checkH1ShouldBeOneTest(FirstTest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkH1ShouldBeOneTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:24 EEST" name="tearDown" finished-at="2024-04-02T11:46:24 EEST" duration-ms="126" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:24 EEST" name="setUp" finished-at="2024-04-02T11:46:28 EEST" duration-ms="3391" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="checkShouldNotHaveHorizontalScrollbar()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:28 EEST" name="checkShouldNotHaveHorizontalScrollbar" finished-at="2024-04-02T11:46:28 EEST" duration-ms="15" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkShouldNotHaveHorizontalScrollbar -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:28 EEST" name="tearDown" finished-at="2024-04-02T11:46:28 EEST" duration-ms="179" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:28 EEST" name="setUp" finished-at="2024-04-02T11:46:31 EEST" duration-ms="3079" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="getAllUrl()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:31 EEST" name="getAllUrl" finished-at="2024-04-02T11:46:34 EEST" duration-ms="2978" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllUrl -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:34 EEST" name="tearDown" finished-at="2024-04-02T11:46:34 EEST" duration-ms="145" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:34 EEST" name="setUp" finished-at="2024-04-02T11:46:37 EEST" duration-ms="3007" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isPresentContactUsPage()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:37 EEST" name="isPresentContactUsPage" finished-at="2024-04-02T11:46:39 EEST" duration-ms="2136" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [vhvhj] but found [КОНТАКТИ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [vhvhj] but found [КОНТАКТИ]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at FirstTest.isPresentContactUsPage(FirstTest.java:70)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isPresentContactUsPage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:39 EEST" name="tearDown" finished-at="2024-04-02T11:46:39 EEST" duration-ms="160" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:39 EEST" name="setUp" finished-at="2024-04-02T11:46:42 EEST" duration-ms="2951" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isPresentLorenIpsonTest()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:46:42 EEST" name="isPresentLorenIpsonTest" finished-at="2024-04-02T11:49:08 EEST" duration-ms="145766" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertFalse(Assert.java:78)
at org.testng.Assert.assertFalse(Assert.java:88)
at FirstTest.isPresentLorenIpsonTest(FirstTest.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isPresentLorenIpsonTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:08 EEST" name="tearDown" finished-at="2024-04-02T11:49:08 EEST" duration-ms="132" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:08 EEST" name="setUp" finished-at="2024-04-02T11:49:11 EEST" duration-ms="3176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isPresentSearchPageTest()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:11 EEST" name="isPresentSearchPageTest" finished-at="2024-04-02T11:49:15 EEST" duration-ms="3237" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isPresentSearchPageTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:15 EEST" name="tearDown" finished-at="2024-04-02T11:49:15 EEST" duration-ms="135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:15 EEST" name="setUp" finished-at="2024-04-02T11:49:18 EEST" duration-ms="3230" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="isPresentValueAttributeAltImg()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:18 EEST" name="isPresentValueAttributeAltImg" finished-at="2024-04-02T11:49:19 EEST" duration-ms="1010" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at FirstTest.isPresentValueAttributeAltImg(FirstTest.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- isPresentValueAttributeAltImg -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FirstTest@3db64bd4]" started-at="2024-04-02T11:49:19 EEST" name="tearDown" finished-at="2024-04-02T11:49:19 EEST" duration-ms="200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- FirstTest -->
      <class name="VisualTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:VisualTest@253b380a]" started-at="2024-04-02T11:44:05 EEST" name="setUp" finished-at="2024-04-02T11:44:08 EEST" duration-ms="3290" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWebsiteLayout()[pri:0, instance:VisualTest@253b380a]" started-at="2024-04-02T11:44:08 EEST" name="testWebsiteLayout" finished-at="2024-04-02T11:44:09 EEST" duration-ms="1251" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWebsiteLayout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:VisualTest@253b380a]" started-at="2024-04-02T11:44:09 EEST" name="tearDown" finished-at="2024-04-02T11:44:09 EEST" duration-ms="159" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- VisualTest -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite1 -->
</testng-results>
